### ISSUE TITLE
Certificate Error when trying to use Biopython within DJango 3.2.12

### ISSUE THREAD

**Issue by DeadM8** [biopython/biopython#3891]
### Setup

I am reporting a problem with Biopython version, Python version, and operating
system as follows:

CPython
Windows-10-10.0.19041-SP0
1.79

### Expected behaviour

I am able to run Bio.ExPASy from within DJango

### Actual behaviour

When I try to run any code from with DJango which contains a call to Biopython, in this case, Bio.Expasy.ScanProsite, I get the following certificate error:

`urllib.error.URLError: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1123)>`

### Steps to reproduce

Create a DJango application, and use a button to start a function which contains a call to the Biopython API.

**Comment by peterjc** [comment]
I don't think this is a Biopython specific issue, but rather a fairly common Python issue to do with the basic SSL certificates it has for using HTTPS links. It may depend on how you installed Python.

What version of Python do you have, and how was it installed (e.g. via conda, via a windows Installer, via WSL, etc)?

**Comment by DeadM8** [comment]
> I don't think this is a Biopython specific issue, but rather a fairly common Python issue to do with the basic SSL certificates it has for using HTTPS links. It may depend on how you installed Python.
> 
> What version of Python do you have, and how was it installed (e.g. via conda, via a windows Installer, via WSL, etc)?

Thank you for your quick response.

Apologies if it's not a Biopython specific question, and thanks for helping me with it anyway.

I'm using Python3.9.4, which was installed via Conda.

Thanks!
